src/utils/stringFormater.ts

```diff
@@ -0,0 +1,10 @@
+export const snakeToCapitalization = (item: string) => {
+  return item
+    .split("_")
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(" ");
+};
+
+export const capitalization = (item: string) => {
```

Comment:
@muhammad-bilal-ali-khokhar  why we can’t fix this with css?

Reply:
Good call Nadir.

Reply:
css capitalize only changes the visual appearance, not the actual string value.when the capitalized value is needed programmatically, a function is required , in some case i need to convert values into function thats why i have created this function